bin/upload_buildkite_test_analytics_junit

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -eu
+
+FILE_PATH=$1
+ANALYTICS_TOKEN=$2
+
+if [ -z "$FILE_PATH" ]; then
+	echo "You must pass a path to the file you want to upload to Test Analytics"
+	exit 1
+fi
+
+if [ -z "$ANALYTICS_TOKEN" ]; then
+	echo "You must pass a Buildkite Analytics Token"
+	exit 1
+fi
+
+if [ ! -f "$FILE_PATH" ]; then
+	echo "No file exists at $FILE_PATH. Cancelling Upload"
+	exit 0
+fi
+
+echo "Uploading ${FILE_PATH} to Test Analytics"
+
+curl -X POST \
+  -H "Authorization: Token token=$ANALYTICS_TOKEN" \
+  -F "format=junit" \
+  -F "data=@$FILE_PATH" \
+  -F "run_env[CI]=buildkite" \
+  -F "run_env[key]=$BUILDKITE_BUILD_ID" \
+  -F "run_env[number]=$BUILDKITE_BUILD_NUMBER" \
+  -F "run_env[job_id]=$BUILDKITE_JOB_ID" \
+  -F "run_env[branch]=$BUILDKITE_BRANCH" \
+  -F "run_env[commit_sha]=$BUILDKITE_COMMIT" \
+  -F "run_env[message]=$BUILDKITE_MESSAGE" \
+  -F "run_env[url]=$BUILDKITE_BUILD_URL" \
+  https://analytics-api.buildkite.com/v1/uploads
```

Comment:
@oguzkocer left a thoughtful comment and question.

Instead, I'm here to point out this command uses two spaces indentation while the rest of the file uses tabs. 🙃

Reply:
It's a good comment! Fixed in dd80faf.

Now we just need a linter...